api/WorkletGlobalScope.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "WorkletGlobalScope": {
+      "__compat": {
+        "spec_url": "https://html.spec.whatwg.org/multipage/worklets.html#workletglobalscope",
+        "support": {
+          "chrome": {
+            "version_added": "65"
```

Comment:
Confirmed by https://storage.googleapis.com/chromium-find-releases-static/9df.html#9df518ea41271fef3a3b4b9b85164b632092c5db, this was enabled together with the Custom Paint API and matches this:
https://github.com/mdn/browser-compat-data/blob/main/api/PaintWorkletGlobalScope.json